docs/USAGE.rst

```diff
@@ -115,17 +115,17 @@ An image is copied into the ``Original`` folder before it will be redacted and w
 Step 4 Users Approve Deidentified Files
 ---------------------------------------
 
-Images in the ``Redacted`` folder have gone through the redaction process, but should be inspected to determine if they still contain PHI/PII or are fully cleared and ready for release.
+Images in the ``Redacted`` folder have gone through the redaction process but should be inspected to determine if they still contain PHI/PII or are fully cleared and ready for release.
 
 Once an image is in the ``Redacted`` folder, the user can approve the file. The user can make the following UI selections:
 
-- "Approve" to approve it, once it has been fully cleared for release. If this is pressed, then the image will move to the ``Approved`` folder and then the view will change to the next image to be processed, as if you had clicked on the ``Next Item`` action in the left menu.
+- "Approve" to approve it, once it has been fully cleared for release. If this is pressed, then the image will move to the ``Approved`` folder, and then the view will change to the next image to be processed, as if you had clicked on the ``Next Item`` action in the left menu.
 - "Quarantine" for more reprocessing, sending it to the ``Quarantined`` folder (go to step 4a)
 
 Step 4a Users Contact IMS and Reprocess for Potential Resolution
 ----------------------------------------------------------------
 
-The ``Quarantined`` folder is for holding images that may hold PHI/PII. Users should contact IMS for any WSI files in the ``Quarantined`` folder, and then review files for reprocessing based on instructions from IMS. If PHI/PII is seen in an image or metadata field, that is somewhere other than the ``Available To Process`` folder, it should be quarantined for reprocessing.
+The ``Quarantined`` folder is for holding images that may hold PHI/PII. Users should contact IMS via Squish (https://www.squishlist.com/seerdms/support/) for any WSI files in the ``Quarantined`` folder, and then review files for reprocessing based on instructions from IMS. If PHI/PII is seen in an image or metadata field, that is somewhere other than the ``Available To Process`` folder, it should be quarantined for reprocessing. For definitions of PHI/PII categories available for recaction, refer to the PHI/PII Definitions and Types section of these usage instructions below.
```

Comment:
Do you want to make this an rst link?  ``via Squish (`https://www.squishlist.com/seerdms/support/ <https://www.squishlist.com/seerdms/support/>`__)``

Should the `PHI/PII Definitions and Types section` also be a link? `` `PHI/PII Definitions and Types section <#phipii-definitions-and-types>`__``?